articles/architecture-scenarios/implementations/mobile-api/mobile-implementation-android.md

```diff
@@ -0,0 +1,1260 @@
+---
+description: The Android implementation of the API for the Mobile + API architecture scenario
+url: /architecture-scenarios/application/mobile-api/mobile-implementation-android
+toc: true
+---
+
+# The Mobile Implementation in Android
+
+This document is part of the [Mobile + API Architecture Scenario](/architecture-scenarios/application/mobile-api) and it explains how to implement the Mobile application in Android. Please refer to the scenario for information on the implemented solution.
+
+::: note
+The full source code for the Node.js API implementation can be found in [this GitHub repository](https://github.com/auth0-samples/auth0-pnp-abc-timesheets/tree/master/timesheets-mobile/android).
+:::
+
+## 1. Set Up the Application
+
+### System Requirements
+
+- [Android Studio 2.3.3](https://developer.android.com/studio/index.html)
+- [Nexus 5X virtual device running Marshmallow (API 23) Android 6.0](https://developer.android.com/studio/run/managing-avds.html)
+
+### Set the Dependencies
+
+For this implementation, we will use the following dependencies within the app’s `build.gradle` file:
+
+- __[Auth0.Android](https://github.com/auth0/Auth0.Android)__: this package enables integration with Auth0 to authenticate users.
+- __[OkHttp](http://square.github.io/okhttp/)__: this package provides an HTTP client to make requests to the Node.JS API.
+- __[JWTDecode.Android](https://github.com/auth0/JWTDecode.Android)__: this package will assist with decoding JWTs.
+- __AppCompat__: this package lets us use the toolbar widget for navigation in our activities.
+
+```gradle
+dependencies {
+    compile 'com.squareup.okhttp:okhttp:2.7.5'
+    compile 'com.auth0.android:auth0:1.10.0'
+    compile 'com.auth0.android:jwtdecode:1.1.1'
+    compile 'com.android.support:appcompat-v7:25.3.1'
+    testCompile 'junit:junit:4.12'
+}
+```
+
+### Update the Manifest
+
+Open the application's `AndroidManifest.xml` and add the internet permission:
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+We’ll also update the application details to utilize the Toolbar widget:
+
+```xml
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.AppCompat.Light.NoActionBar">
+    </application>
+```
+
+### Set Configuration Values
+
+Set your Auth0 Client ID, Auth0 Domain, and API’s url in the `strings.xml` resource:
```

Comment:
Do we explain anywhere where the user can find these values? If not maybe add a note?